.github/workflows/build-pr-artifacts.yml

```diff
@@ -0,0 +1,124 @@
+name: Build Artifacts for PRs
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+env:
```

Comment:
For hotifx branches, does it go under `develop-rudder-transformer` ?

Reply:
Yes, for any PR in general.